Update formatting docs to remove soon-to-be deprecated settings (#6476)
microsoft/vscode-docs: docs/python/editing.md

### Setting a default formatter
Once you install a formatter extension, you can select it as the default formatter for Python files in VS Code by following the steps below:
1. Open a Python file in VS Code
1. Right click on the editor
1. Select **Format Document With...**
1. Select **Configure Default Formater...** from the drop-down menu
1. Select your preferred formatter extension from the list.

Alternatively, you can set it as the default formatter for all Python files by setting `"editor.defaultFormatter"` in your User `settings.json` file, under a `[python]` scope.

For example, to set Black Formatter as the default formatter, add the following setting to your User `settings.json` file:

```
  "[python]": {
    "editor.defaultFormatter": "ms-python.black-formatter"
  }
```

### Formatting your code

You can format your code by right-clicking on the editor and selecting **Format Document**, or by using the `kb(editor.action.formatDocument)` keyboard shortcut.

> You can also set up your code to be formatted on save by adding the following setting to your User `settings.json` file:

Reviewer: ```suggestion
You can add the following setting to your User `settings.json` file to enable formatting on save for your code:
```
